CODING_STANDARDS.md

```diff
@@ -37,14 +37,21 @@ The commit messages are typically rendered in [Markdown format](https://docs.git
 When updating your code, please make sure you perform a rebase, not a merge with the latest branch.
 Pull requests should be a simple fast-forward of the branch they are intended to land on.
 
-The correct way to rebase (if working on top of 10.3 branch):
+The correct way to rebase (if working on top of 10.4 branch):
 
 ```sh
-git fetch upstream/10.3  # This assumes upstream is github.com/MariaDB/server
-git rebase upstream/10.3
+git fetch upstream/10.4  # This assumes upstream is github.com/MariaDB/server
+git rebase upstream/10.4
```

Comment:
10.4 reaches EOL in half a year. You might want to use a later branch here, say, 10.11.

Reply:
Done